pkgs/by-name/aw/awscli2/package.nix

```diff
@@ -118,6 +118,11 @@ py.pkgs.buildPythonApplication rec {
     less
   ];
 
+  makeWrapperArgs = [
+    "--unset NIX_PYTHONPATH"
```

Comment:
Each argument should be a separate string (to avoid issues when templating or escaping strings).